Assets/Scripts/GameData/Creatures/ICharacter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using SwordAndBored.GameData.Abilities.Skills;
+using SwordAndBored.GameData.Equipment;
+using SwordAndBored.GameData.Roles;
+
+namespace SwordAndBored.GameData.Creatures
+{
+    /// <summary>
+    /// A unit that has specific equipment slots and a progressing skill tree
+    /// </summary>
+    interface ICharacter : ICreature
+    {
+        IRole Role { get; }
+        int XP { get; set; }
+        int Level { get; }
+        ISkillTree SkillTree { get; }
+        IEquipment ArmorSlot { get; set; }
+        IEquipment WeaponSlot { get; set; }
+        IEquipment AdditionalEquipmentSlot { get; set; }
```

Comment:
It is highly possible that more than 1 additional equipment could be worn by a character at a time. I would suggest replacing this field with an array of IEquipment with an arbitrary length such as 4 or 6.